tests/plugins/clipboard/manual/dndnotificationoff.html

```diff
@@ -0,0 +1,32 @@
+<h2>Classic editor</h2>
+
+<button id="turnNotifyOff" style="margin-bottom: 20px;">Turn notification off</button>
+
+<div id="editor">
+	<p>Drag and drop file here.</p>
+</div>
+
+<h2>Inline editor</h2>
+<div id="inline" contenteditable="true" style="border:1px solid #000000;">
+	<p>Drag and drop file here.</p>
+</div>
+
+<script>
+	var editor = CKEDITOR.replace( 'editor', {
+			language: 'en'
+		} ),
+		inline = CKEDITOR.inline( 'inline', {
+			language: 'en'
+		} );
+
+	editor.on( 'loaded', function() {
+		if( !CKEDITOR.plugins.clipboard.isFileApiSupported || bender.tools.env.mobile ){
+			return bender.ignore();
+		}
+
+		var btn = document.getElementById( 'turnNotifyOff' );
+		btn.addEventListener( 'click', function() {
+			CKEDITOR.plugins.clipboard.ignoreUnsupportedMimeTypeNotification = true;
```

Comment:
The same as in https://github.com/ckeditor/ckeditor4/pull/5134/files/72c55fff0099f399e026189b94e8016791e733d4#r854003585. leftover and probably duplicated test.